blueman/gui/manager/ManagerDeviceMenu.py

```diff
@@ -168,13 +168,32 @@ def on_device_property_changed(self, lst: "ManagerDeviceList", _device: Device,
             if key in ("Connected", "UUIDs", "Trusted", "Paired"):
                 self.generate()
 
+    def _translate_error_message(self, msg: str) -> str:
+        msg = msg.split(":", 3)[-1].strip()
+
+        # https://sourceware.org/git/?p=glibc.git;a=blob;f=sysdeps/gnu/errlist.h
+        # https://git.musl-libc.org/cgit/musl/tree/src/errno/__strerror.h
+        # https://git.uclibc.org/uClibc/tree/libc/string/_string_syserrmsgs.c
+        if msg == "Protocol not available":
+            # ENOPROTOOPT
+            return _("No audio endpoints registered")
```

Comment:
If I am not mistaken we connect a single endpoint not multiple.

Reply:
pulseaudio registers at least a source and a sink endpoint with BlueZ. Anyway, I'm actually not sure how much better this is to the user than "Protocol not available". Maybe something like "No Bluetooth audio support registered"?

Reply:
I forgot we have a source and a sink, so this lgtm 😄 .